Makefile

```diff
@@ -130,11 +130,10 @@ gopkg:
 
 # Build a release tarball
 charmstore-$(VERSION).tar.xz: $(VERSIONDEPS)
-	mkdir -p charmstore-$(VERSION)/bin
-	GOBIN=$(CURDIR)/charmstore-$(VERSION)/bin go install $(INSTALL_FLAGS) -v $(PROJECT)/...
-	mv charmstore-$(VERSION)/bin/charmd charmstore-$(VERSION)/bin/charmstore
-	tar cv charmstore-$(VERSION) | xz > $@
-	-rm -r charmstore-$(VERSION)
+	mkdir -p charmstore-release/bin
+	GOBIN=$(CURDIR)/charmstore-release/bin go install $(INSTALL_FLAGS) -v $(PROJECT)/...
+	tar cv -C charmstore-release . | xz > $@
```

Comment:
Not sure I'd bother with the "v" *shrug*. Then again I wouldn't bother with the "-v" on the go install either. Just taste I guess. *shrug*